### PR TITLE
TAV-191B: Add query limits to prevent service timeouts & OOME

### PR DESCRIPTION
**Pull request by ssaleem-ee**

### CODE REVIEW

File: app/Module.scala [hunk 1]
```diff
@@ -60,6 +60,17 @@ class Module(environment: Environment,
     ElasticSettings(localMode, homeDir, preDelete, connectionString, isCluster, clusterName, numShards)
   }
 
+  @Provides
+  @Singleton
+  def provideRdsQueryConfig(configHelper: ConfigHelper): RdsQueryConfig = {
+    val queryTimeoutMillis =
+      configHelper.getConfigString("address-lookup-rds.query-timeout-ms").map(_.toInt).getOrElse(10000)
```
Comment (Ardesco) [inline app/Module.scala:67]: Should we parameterise these and put them in config to make it easy to tweak in the future?

File: app/Module.scala [hunk 2]
```diff
@@ -60,6 +60,17 @@ class Module(environment: Environment,
...
+    val queryResultsLimit =
+      configHelper.getConfigString("address-lookup-rds.query-results-limit").map(_.toInt).getOrElse(300)
```
Comment (Ardesco) [inline app/Module.scala:69]: As above

File: app/Module.scala (hunk 1 above)
Comment (Ardesco) [reply app/Module.scala:67]: actually looks like they are in app conf, just not used

**Review by Ardesco — CHANGES_REQUESTED** [review]

File: app/Module.scala (hunk 1 above)
Comment (ssaleem-ee) [reply app/Module.scala:67]: oh - what makes you say that the application.conf values aren't being used?

**Review by Ardesco — APPROVED** [review]

**Pull request merged**